ts/main.ts

```diff
@@ -171,5 +211,44 @@ const $newEntryAnchor = document.querySelector('.new-entry-anchor');
 if (!$newEntryAnchor) throw new Error('.new-entry-anchor query failed!');
 $newEntryAnchor.addEventListener('click', function () {
   viewSwap('entry-form');
+  $entryFormTitle.textContent = 'New Entry';
+  $urlInput.value = '';
+  $titleInput.value = '';
+  $previewImg.setAttribute('src', '../images/placeholder-image-square.jpg');
+  $notes.value = '';
+  data.editing = null;
   writeData();
 });
+
+const $entryFormTitle = document.querySelector(
+  '.entry-form-title',
+) as HTMLHeadingElement;
+if (!$entryFormTitle) throw new Error('.entry-form-title query failed!');
+
+$entriesList.addEventListener('click', function (event: Event) {
+  const eventTarget = event.target as HTMLElement;
+  console.log('eventTarget:', eventTarget);
+  console.log('eventTarget.tagName:', eventTarget.tagName);
+  if (eventTarget.tagName === 'I') {
```

Comment:
Line 232 you should change this to be a "guard clause." Instead of running the code _if_ the tagName is 'I', we should exit anytime it is _not_ 'I'. This will be a little more efficient and more readable for the rest of your function.